example/spec/serial/DiscoverGranulesHttpSpec.js

```diff
@@ -190,54 +181,11 @@ describe('The Discover Granules workflow with http Protocol', () => {
       ingestGranuleExecutionStatus = await waitForCompletedExecution(ingestGranuleWorkflowArn1);
     });
 
-    afterAll(async () => {
```

Comment:
👍🏻  for posterity this is because the after all for the entire test also deletes these, and deletions with a default throw will now cause this to fail.